vllm/model_executor/layers/quantization/torchao_utils.py

```diff
@@ -0,0 +1,60 @@
+# SPDX-License-Identifier: Apache-2.0
+"""
+Common utilities for torchao.
+"""
+
+from typing import Any, Dict, Set
+
+import torch
+
+
+def torchao_quantize_param_data(param: torch.Tensor,
+                                torchao_config: Any) -> torch.nn.Parameter:
+    """Quantize a Tensor with torchao quantization specified by torchao_config
+
+    Args:
+       `param`: weight parameter of the linear module
+       `torchao_config`: type of quantization and their arguments we want to
+        use to quantize the Tensor
+    """
+    try:
+        from torchao.core.config import AOBaseConfig
```

Comment:
IMO we should move `AOBaseConfig` to `torchao/__init__.py` and import from there